src/sources/splunk_hec.rs

```diff
@@ -0,0 +1,954 @@
+use crate::{
+    event::{self, Event, LogEvent, ValueKind},
+    topology::config::{DataType, GlobalOptions, SourceConfig},
+};
+use bytes::{buf::IntoBuf, Bytes};
+use chrono::{TimeZone, Utc};
+use flate2::read::GzDecoder;
+use futures::{future, sync::mpsc, Async, Future, Sink, Stream};
+use hyper::service::Service;
+use hyper::{header::HeaderValue, Body, Method, Request, Response, Server, StatusCode};
+use lazy_static::lazy_static;
+use serde::{de, Deserialize, Serialize};
+use serde_json::{de::IoRead, json, Deserializer, Value};
+use std::sync::{Arc, Mutex};
+use std::{
+    io::Read,
+    net::{Ipv4Addr, SocketAddr},
+};
+use stream_cancel::{Trigger, Tripwire};
+use string_cache::DefaultAtom as Atom;
+
+// Event fields unique to splunk_hec source
+lazy_static! {
+    pub static ref CHANNEL: Atom = Atom::from("channel");
+    pub static ref INDEX: Atom = Atom::from("index");
+    pub static ref SOURCE: Atom = Atom::from("source");
+    pub static ref SOURCETYPE: Atom = Atom::from("sourcetype");
+}
+
+/// Cashed bodies for common responses
```

Comment:
```suggestion
/// Cached bodies for common responses
```

is this what you meant?

Reply:
Yep